src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -133,7 +134,9 @@ def __init__(
         self._variant_id = variant_id
         self._validate_inputs = validate_inputs
         self._worker_count = flow_executor._worker_count
-        use_fork = multiprocessing.get_start_method() == "fork"
+        self.multiprocessing_start_method = os.environ.get("PF_BATCH_METHOD")
```

Comment:
Do we need to store this as an object property?

Reply:
![image](https://github.com/microsoft/promptflow/assets/115983968/b8d55435-1ed2-49f3-91b7-8e78f83b94bb)
Yes, save below

Reply:
fixed